app/src/main/java/com/example/babytracking/BreastActivity.kt

```diff
@@ -1,11 +1,136 @@
 package com.example.babytracking
 
 import android.os.Bundle
+import android.text.TextUtils
+import android.view.View
+import android.widget.AdapterView
+import android.widget.ArrayAdapter
+import android.widget.TimePicker
+import android.widget.Toast
 import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.breast_layout.breast_quality_baby
+import kotlinx.android.synthetic.main.breast_layout.breast_button_save
+import kotlinx.android.synthetic.main.breast_layout.breast_length
+import kotlinx.android.synthetic.main.breast_layout.breast_quality_mother
+import kotlinx.android.synthetic.main.breast_layout.spinner_breast
```

Comment:
While layout IDs typically use snake_case, the convention is to use camelCase for views that you will access in code by their IDs using Kotlin. This way, you keep your classes more Android-y.